course-app/src/app/app.component.ts

```diff
@@ -7,5 +7,34 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
     public userName: string = 'Harry Potter';
-    public infoTitle: string = 'Your List Is Empty'
+    public infoTitle: string = 'Your List Is Empty';
+
+    private isCurrentPage: CurrentPage = CurrentPage.Main;
+
+    public isLoginPage: boolean = this.isCurrentPage === CurrentPage.Login;
```

Comment:
better to do it as getters
Example:
public get isLoginPage(): boolean {
    return this.isCurrentPage === CurrentPage.Login
}